### PR TITLE
Build documentation again with Travis

### PR DESCRIPTION
**Pull request by amontoison**

### CODE REVIEW

**Comment by abelsiqueira** [comment]
It's failing because of the breakage tests. #241 is the fix.

**Comment by abelsiqueira** [comment]
#241 is merged, can you rebase this?

File: .travis.yml [hunk 1]
```diff
@@ -28,6 +24,8 @@ branches:
     - /^v\d+\.\d+(\.\d+)?(-\S*)?$/ # tags
 
 jobs:
+  allow_failures:
+    - julia: nightly
```
Comment (abelsiqueira) [inline .travis.yml]: In line 34, please add `AND fork = false`. I'm not sure why my tests did not run for this, but yours do.

Comment (amontoison) [reply .travis.yml]: Great, It works now.

**Comment by dpo** [comment]
Thank you.

**Pull request merged**